eng/Version.Details.xml

```diff
@@ -0,0 +1,367 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Dependencies>
+  <ProductDependencies>
+    <Dependency Name="Microsoft.NET.Sdk.Razor" Version="3.0.0-preview-19057-06">
+      <Uri>https://github.com/aspnet/AspNetCore-Tooling</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="dotnet-ef" Version="3.0.0-preview.18604.3">
+      <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore.Design" Version="3.0.0-preview.18604.3">
+      <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore.InMemory" Version="3.0.0-preview.18604.3">
+      <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore.Relational" Version="3.0.0-preview.18604.3">
+      <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore.Sqlite" Version="3.0.0-preview.18604.3">
+      <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore.SqlServer" Version="3.0.0-preview.18604.3">
+      <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore.Tools" Version="3.0.0-preview.18604.3">
+      <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
+      <Sha>000000</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.EntityFrameworkCore" Version="3.0.0-preview.18604.3">
```

Comment:
Random thought: You could just list this one from EF and use $(MicrosoftEntityFrameworkCoreVersion) everywhere.

Reply:
I'd rather have a fully-defined dependency list. This will help us when doing static analysis for the impacts of changes like package deprecation, rename, one-off-builds, etc.